internal/daemon/worker/testing.go

```diff
@@ -532,9 +537,10 @@ func NewTestMultihopWorkers(t testing.TB,
 		WorkerAuthKms:              childDownstreamWrapper2,
 		WorkerAuthDebuggingEnabled: enableAuthDebugging,
 	})
+	t.Cleanup(childKmsWorker.Shutdown)
 
 	// Sleep so that workers can startup and connect.
-	time.Sleep(10 * time.Second)
+	time.Sleep(12 * time.Second)
```

Comment:
Out of curiosity, I was wondering if there are other ways to check if things are connected rather than using sleeps. Is there a way to check the worker's status and verify that it's connected?

Reply:
I had that thought too but (a) don't want to spend the time now and (b) it's not clear to me for all dependent tests what metric would be correct. On some ent tests for instance "connected" can mean something very different. So probably it's best up to the caller of this function to do that verification.